src/Core/LeanCode.Time/LeanCode.Time.csproj

```diff
@@ -10,6 +10,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Update="System/Strings.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Strings.resx</DependentUpon>
+    </Compile>
```

Comment:
Ah, so this is the magic invocation I was missing so the compiler doesn't complain about this file being included twice? Good to know.

Reply:
Nope, that's not it unfortunately. I added it to test that and left it just in case.